lib/src/sdk/trace/tracer_provider.dart

```diff
@@ -27,14 +27,19 @@ class TracerProviderBase implements api.TracerProvider {
   @protected
   final sdk.SpanLimits spanLimits;
 
+  @protected
+  final sdk.TimeProvider _timeProvider;
```

Comment:
Because the member is private, does it make sense to remove `@protected`?

```suggestion
  final sdk.TimeProvider _timeProvider;
```